libopenage/renderer/render_pass.h

```diff
@@ -14,6 +14,18 @@ namespace openage {
 namespace renderer {
 class RenderTarget;
 
+/**
+ * Stencil states for the render pass.
+ */
+enum class StencilState {
+	/// State for writing GUI elements to stencil buffer.
+	WRITE_STENCIL_MASK,
+	/// State for using the mask when rendering scene.
+	USE_STENCIL_TEST,
+	/// State for normal rendering (GUI rendering).
+	DISABLE_STENCIL
```

Comment:
These docstrings refer to the GUI, but stencil testing should not just be used for this purpose. So it's better to make the docstrings more agnostic.